rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -45,6 +45,10 @@ foreach(_extension IN LISTS AMENT_EXTENSIONS_rosidl_generate_interfaces)
   endif()
 endforeach()
 
+if (_typesupport_impls STREQUAL "" )
```

Comment:
Shouldn't this be like this? My CMake-fu is pretty bad.

`if ("${_typesupport_impls} " STREQUAL " ")`